### ISSUE TITLE
Not able to replicate the results on 4bit quantised LLaMA 3.2 3B on prontoqa task.

### ISSUE THREAD

**Issue by 1609abhi** [Yu-Fangxu/FoR#10]
We finetuned LLaMA 3.2 3B using 4bit =True. However when tested against LLaMA 3.2 3B pre-trained the performance didn't improve.

**Comment by Yu-Fangxu** [comment]
Hi,
Could you please try Llama-3-8B? I'm not sure if the 3B model is sufficient to reason on PronptoQA

**Comment by 1609abhi** [comment]
Hi,
I tried LLaMA-3-8B also, but was not able to replicate the results through
it also. I used 4 bit quantisation only. Also I found several discrepancy
regarding the validation/test set in your code. Particularly the splits
were different from what was claimed in the paper.
